content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -41,7 +41,7 @@ the nodes, managing the pods and containers running on a machine. The kubelet
 translates each pod into its constituent containers and fetches individual 
 container usage statistics from the container runtime through the container 
 runtime interface. The kubelet fetches this information from the integrated 
-cAdvisor for the legacy Docker integration.  It then exposes the aggregated pod 
+cAdvisor for cgroup-based container runtimes.  It then exposes the aggregated pod 
```

Comment:
Your suggestion LGTM Tim. I reworded this based on [my question in #sig-node](https://kubernetes.slack.com/archives/C0BP8PW9G/p1645657340951729) if that helps

Reply:
There is a KEP scheduled to move to beta in 1.24 to not use cAdvisor for metric collection and to use the CRI implementation
Perhaps this can be part of the documentation change for 1.24 for this KEP
[SIG Node KEP 2371 cAdvisor-less, CRI-full Container and Pod Stats](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2371-cri-pod-container-stats)
/sig node

cc @nate-double-u

Reply:
I like that idea Rey. So it's expanding the CRI summary API to rely on the CRI implementation for Pod and container stats instead of relying on cAdvisor? Cool! What would I need to do here? Change the destination branch to dev-1.24?

Reply:
@reylejano if you want we can discuss this on Slack and I'm happy to get involved in the docs changes for this KEP for 1.24. LMK :)